README.md

```diff
@@ -431,6 +431,14 @@ Depends on configuration, but usually it is not needed to generate or use API ke
 ## 12. Registering new LLM provider
    Please look [here](https://github.com/openshift/lightspeed-service/blob/main/CONTRIBUTING.md#adding-a-new-providermodel) for more info.
 
+## 13. Fine tuning
+   Service uses so called system prompt to put the question into context before the question is send to selected LLM. The default system prompt is fine tuned for questions about OpenShift and Kubernetes. It is possible to use different system prompt via the configuration option `system_prompt_path` in section `ols_config`. That option must contain path to text file with actual system prompt (can contain multiple lines). An example of such configuration:
```

Comment:
The service uses **the,** so called**,** system prompt to...

Reply:
... before the question is **sent** to **the** selected LLM

Reply:
It is possible to use **a** different system prompt

Reply:
... in **the** `ols_config` section

Reply:
That option must contain **the** path to **the** text file

Reply:
fixed, thank you